db/models/round.py

```diff
@@ -76,3 +76,9 @@ class Round(BaseModel):
         default=False,
         nullable=False,
     )
+    mark_as_complete_enabled = Column(
```

Comment:
we are adding too many columns in round table related to round configuration. Going forward, we need to have a json config column for all the round related config information.

Reply:
Yeah I think that's probably a good shout. I'd like to leave this as a column for now though as Steve wanted it in by tomorrow but I'll raise a ticket to look at this.